src/index.js

```diff
@@ -11,6 +11,7 @@ import ThemeProvider from 'components/ThemeProvider';
 
 import './styles/index.scss';
 import StaticApp from 'components/StaticApp';
+import { AuthModalProvider } from 'components/Auth/AuthModalProvider';
```

Comment:
any reason `AuthModalProvider` export isn't added to `Auth/index.ts`?  I don't have a strong opinion on it, but some of the components in the directory are included and some aren't

Reply:
Nope - added to `Auth/index.ts`